docs/getting-started/prefect-quickstart.md

```diff
@@ -0,0 +1,167 @@
+18:21:40.837 | INFO    | Flow run 'fine-gorilla' - Created task run 'get_repo_info-0' for task 'get_repo_info'
+18:21:40.838 | INFO    | Flow run 'fine-gorilla' - Executing 'get_repo_info-0' immediately...
+18:21:41.468 | INFO    | Task run 'get_repo_info-0' - Finished in state Completed()
+18:21:41.477 | INFO    | Flow run 'fine-gorilla' - Stars 🌠 : 12340
+18:21:41.606 | INFO    | Flow run 'fine-gorilla' - Created task run 'get_contributors-0' for task 'get_contributors'
+18:21:41.607 | INFO    | Flow run 'fine-gorilla' - Executing 'get_contributors-0' immediately...
+18:21:42.225 | INFO    | Task run 'get_contributors-0' - Finished in state Completed()
+18:21:42.232 | INFO    | Flow run 'fine-gorilla' - Number of contributors 👷: 30
+18:21:42.383 | INFO    | Flow run 'fine-gorilla' - Finished in state Completed('All states completed.')
+```
+</div>
+
+
+You'll find a link directing you to the flow run page conveniently positioned at the top of your flow logs.
+![Alt text](flow-run-diagram.jpg)
+
+Local flow run execution is great for development and testing, but in order to schedule flow runs or trigger them based on events, you’ll need to [deploy](/tutorial/deployments/) your flows.
+
+
+### Step 5: Deploy the flow
+
+Deploying your flows is, in essence, the act of informing the Prefect API of where, how, and when to run your flows.
+
+!!! warning "Always run `prefect deploy` commands from the **root** level of your repo!"
+
+When you run the `deploy` command, Prefect will automatically detect any flows defined in your repository. Select the one you wish to deploy. Then, follow the 🧙 wizard to name your deployment, add an optional schedule, create a work pool, optionally configure remote flow code storage, and more!
+
+<div class="terminal">
+```bash
+prefect deploy
+```
+</div>
+
+!!! note "It's recommended to save the configuration for the deployment."
+    Saving the configuration for your deployment will result in a `prefect.yaml` file populated with your first deployment. You can use this yaml file to edit and [define multiple deployments](/concepts/deployments-ux/) for this repo.
+
+
+### Step 5: Start a Worker and Run Deployed Flow
+
+Start a worker to manage local flow execution. Each worker polls its assigned [work pool](/tutorial/deployments/#why-work-pools-and-workers).
+
+In a new terminal, run:
+<div class="terminal">
+```bash
+prefect worker start --pool '<work-pool-name>'
+```
+</div>
+
+Now that your worker is started, you are ready to kick off deployed flow runs from either the UI or by running:
+
+<div class="terminal">
+```bash
+prefect deployment run '<flow-name>/<deployment-name>'
+```
+</div>
+
+Congrats on your first successfully deployed flow run! 🎉
+
+Now you've seen:
+
+- how to define your flows and tasks using decorators
+- how to deploy a flow
+- how to start a worker
+
+### Next Steps
+
+- For a more detailed explanation of the concepts introduced above, our [tutorial](/tutorial/index/) is recommended. 
```

Comment:
```suggestion
- For a more detailed walk through see the [tutorial](/tutorial/index/). 
```

Reply:
Maybe a little confusing to talk about going to something other than concepts to learn concepts. :)